plugin/al/usdmayautils/AL/usdmaya/utils/MeshUtils.cpp

```diff
@@ -751,8 +759,171 @@ bool MeshImportContext::applyEdgeCreases()
   return false;
 }
 
+
+void MeshImportContext::applyColourSetData()
+{
+  const std::vector<UsdGeomPrimvar> primvars = mesh.GetPrimvars();
+  for (auto it = primvars.begin(), end = primvars.end(); it != end; ++it)
+  {
+
+    const UsdGeomPrimvar& primvar = *it;
+    TfToken name, interpolation;
+    SdfValueTypeName typeName;
+    int elementSize;
+
+    primvar.GetDeclarationInfo(&name, &typeName, &interpolation, &elementSize);
```

Comment:
Not really sure why we creating `typeName `and `elementSize ` objects that are not used?
Can't just pass nullptr for the arguments we don't need to be populated?
`primvar.GetDeclarationInfo(&name, nullptr , &interpolation, nullptr );`